Add files via upload

ReeseOchitwa opened the pull request:
Write a short description about your art piece, such as what your artwork represents and your experience drawing it!

My artwork depicts a slightly humanoid cat. I was inspired by my experience of owning six cats in my lifetime. All of them are unique in size, pattern, and shape, but they are adorable nonetheless. Almost all elements of my artwork are based on random ranges of randomized numbers, so I had to be conscious of selecting the correct range limits in order to create a cute or at least funny-looking cat despite the myriad of options for line lengths.


- [x ] I have read the steps to [getting a blot](https://github.com/hackclub/blot/blob/main/docs/GET_A_BLOT.md)
- [ x] I have read all the [submission requirements](https://blot.hackclub.com/submitting) and follow them 
- [x ] I am submitting art that...
  - [x ] is algorithmically generated (will meaningfully change each time the program is run)
  - [ x] is your own work that is not copied from somewhere else
  - [ x] is drawable through the editor(does not use outside tools)
  - [x ] doesn't call `Math.random()` (See the documentation on [randomness](https://github.com/hackclub/blot/blob/main/public/TOOLKIT.md#randomness))
  - [x ] is drawable on Blot itself (doesn't have lines overlap more than 5 times)
    - [x ] Doesn't overlap lines more than 5 times
    - [x ] Fits in the working area of 125x125mm
    - [ x] Doesn't rely on colour to look good

Lastly, link a picture of your art piece here:
![CatAvatar](https://github.com/user-attachments/assets/6507bf11-aea9-437b-84af-f254ea5ee706)

EnterpriseGoose commented:
Hello!
As a fellow cat owner, this is amazing!!! It's so cute and I love hitting run and seeing all the catss!!! :)
To get a blot, though, I'd love to see if you can add a bit more complexity.
Here are some of my ideas:
- You mention in the description that cats come in all sorts of patterns, but the cat you generate has no pattern! Could you figure out having a few different patterns that get randomly chosen/generated?
- Could you make the eyes have different patterns/looks that are randomly chosen/generated? I think that would be super cute!
- The area where the ear intersects with the face sometimes has this extra line segment. Could you figure out how to remove that? 
![image](https://github.com/user-attachments/assets/99be5960-7dca-4f60-8f17-2ff8340b2c86)
(maybe bt.cover or bt.difference would help?)

Overall I think it's super cute and with just a bit more work you'll get a blot! I'm super excited to see what you come up with and to draw tons of little cats and put them everywhere :)

ReeseOchitwa commented:
Hi! Thank you so much for your comments. I took all of your feedback into account and added some more features to my artwork such as two new eye styles, a couple more patterns, and a collar. I also fixed the messed up ear bit. I hope you like it. Thanks!

EnterpriseGoose commented:
It's ADORABLE!!! I love it so much. I'm forwarding this to the main reviewer now. Crossing my fingers you get a blot!
